addons/l10n_co/views/res_partner.xml

```diff
@@ -8,9 +8,6 @@
             <field name="vat" position="before">
                 <field name="l10n_co_document_type"/>
             </field>
-            <field name="vat" position="after">
-                <field name="l10n_co_verification_code"/>
```

Comment:
I would simply hide it, since third-party modules could assume this field to be present in the page (although that's rather unlikely).

Reply:
good point, done